addons/calendar/models/res_partner.py

```diff
@@ -91,6 +91,8 @@ def schedule_meeting(self):
         action = self.env["ir.actions.actions"]._for_xml_id("calendar.action_calendar_event")
         action['context'] = {
             'default_partner_ids': partner_ids,
+            'meetings_parent_partner': self.id,
+            'search_default_invite_partner': 1,
         }
         action['domain'] = ['|', ('id', 'in', self._compute_meeting()[self.id]), ('partner_ids', 'in', self.ids)]
```

Comment:
maybe in some cases, we need domains even if they overlap as there is already some computation to show some records with ids
What do you think?

Reply:
Mm yes I guess it's not too bad in the end, issue was more if the filter changes absolutely nothing.